spyder/plugins/switcher/widgets/switcher.py

```diff
@@ -220,9 +220,7 @@ def _add_item(self, item, last_item=True):
     def clear(self):
         """Remove all items from the list and clear the search text."""
         self.set_placeholder_text('')
-        self.model.beginResetModel()
         self.model.clear()
-        self.model.endResetModel()
```

Comment:
Why is not necessary?

Reply:
See note in commit `Fix warning printed by Qt 6`: `QStandardItemModel.clear() calls beginResetModel() and endResetModel() internally`

(This also applies to Qt 5, and Qt 6 added a warning when `begin/endResetModel` are called in wrong order.)